### ISSUE TITLE
1.14 support?

### ISSUE THREAD

**Issue by redbrain** [MassiveCraft/Factions#1348]
Is there support for 1.14 yet?

**Comment by HorlbogeDE** [comment]
Need that too. Currently waiting to start a minecraft project with this plugin, but the authors dont seem to be very active. I also tried to port it myself, but as I didnt write this plugin its quite hard to figure out how.

**Comment by jaroslawjanas** [comment]
@HorlbogeDE If you port it, please let me know or post it somewhere.
RIP Factions.

**Comment by magnusulf** [comment]
I already have a 1.14 version:
https://github.com/magnusulf/Factions/
https://www.spigotmc.org/threads/factions3-for-1-13.353382/page-5#post-3399603

**Comment by redbrain** [comment]
Yup, running [the latest version of Factions3](https://www.spigotmc.org/resources/factions3-for-1-13.63602/updates) on [the latest spigot](https://getbukkit.org/) seems to work for me, so **look there!**
